fineract-provider/src/main/java/org/apache/fineract/portfolio/interestratechart/api/InterestRateChartSlabsApiResource.java

```diff
@@ -135,7 +122,10 @@ public String retrieveOne(@PathParam("chartId") final Long chartId, @PathParam("
     @POST
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String create(@PathParam("chartId") final Long chartId, final String apiRequestBodyAsJson) {
+    @ApiOperation(value = "Create a Slab", httpMethod = "POST", notes = "Creates a new interest rate slab for an interest rate chart.\n" + "Example Requests:\n" + "\n" + "interestratecharts/1/chartslabs")
```

Comment:
Creats a new interest rate slab for an interest rate chart.

Mandatory Fields
periodType, fromPeriod, annualInterestRate
Optional Fields
toPeriod and description
Example Requests:

interestratecharts/1/chartslabs

Reply:
All the above details in the notes section

Reply:
seperated by new line characters